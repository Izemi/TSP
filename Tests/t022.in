SYD,-33.946111,151.177222
00K,-90.000000,4.000000
DEL,28.559444,77.099444
PPT,-17.553611,-149.607222
