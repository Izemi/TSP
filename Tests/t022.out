-insert   :     37771.14 SYD DEL 00K PPT SYD
